projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas-submit-button/go-off-canvas-submit-button.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'go-off-canvas-submit-button',
+  templateUrl: './go-off-canvas-submit-button.component.html',
+  styleUrls: ['./go-off-canvas-submit-button.component.scss']
+})
+export class GoOffCanvasSubmitButtonComponent {
+  @Input() action: Function;
```

Comment:
Should we handle with this an event? With the portal approach, I think it would be more consistent to use an EventEmitter here. Thoughts?